contracts/liquidators/BalancerLpTokenLiquidator.sol

```diff
@@ -20,25 +20,23 @@ contract BalancerLpTokenLiquidator is IRedemptionStrategy {
     uint256 outputTokenIndex = type(uint256).max;
     address outputTokenAddress = abi.decode(strategyData, (address));
 
+    uint256 offset = 0;
     for (uint256 i = 0; i < tokens.length; i++) {
       if (address(tokens[i]) == outputTokenAddress) {
         outputTokenIndex = i;
         break;
+      } else if (address(tokens[i]) == address(inputToken)) {
+        offset = 1;
       }
     }
 
-    outputToken = tokens[outputTokenIndex];
-
     uint256[] memory minAmountsOut = new uint256[](tokens.length);
     minAmountsOut[outputTokenIndex] = 1;
```

Comment:
`outputTokenIndex` in `minAmountsOut` is not affected by the need to offset, right?

Reply:
Correct! I've verified this